.github/workflows/higgs-shop-deploy-github-pages.yml

```diff
@@ -17,6 +15,8 @@ defaults:
 jobs:
     deploy-github-pages:
         name: Deploy Higgs Shop to Github Pages
+        if: >
```

Comment:
is the > necessary?  since the conditional is just a single line, seems unncessary.